lib/mergeset/table_test.go

```diff
@@ -101,16 +101,30 @@ func TestTableCreateSnapshotAt(t *testing.T) {
 
 	var isReadOnly uint32
 	tb := MustOpenTable(path, nil, nil, &isReadOnly)
-	defer tb.MustClose()
```

Comment:
Comment must be added.

 In case of data ingestion, background workers may hold a reference for inmemoryPartion by setting `inMerge= true` for part. In this case `debugFlush` cannot flush these parts, since it's already in merge by background process.

 There is no way to change this behavior without changing actual logic for background merges. You must stop this process somehow or introduce a lock.